charts/bookkeeper/values.yaml

```diff
@@ -57,7 +57,13 @@ storage:
 jvmOptions:
   memoryOpts: ["-Xms1g", "-XX:MaxDirectMemorySize=2g"]
   gcOpts: []
-  gcLoggingOpts: []
+  ## Unified JVM Logging is used for GC logging from Java 9
+  ## Hence use the following options if the bookkeeper version being deployed
```

Comment:
Can you simplify the comment? Something like this should be enough: 
```
## GC logging for Java 9+
gcLoggingOpts: ["-Xlog:gc*,safepoint::time,level,tags:filecount=5,filesize=64m"]
## GC logging for Java 8
# gcLoggingOpts: ["-XX:+PrintGCDetails","-XX:+PrintGCApplicationStoppedTime","-XX:+UseGCLogFileRotation","-XX:NumberOfGCLogFiles=5","-XX:GCLogFileSize=64m"]
```

Reply:
Sure

Reply:
Done